topics/clang.md

```diff
@@ -827,7 +836,7 @@ failureConditions:
 
 {id="clang-feature-matrix"}
 
-The %qp% linters provide the following %product% features:
+The %qdcpp% family of linters provide the following %product% features:
```

Comment:
The Qodana for C/C++ is not a family of linters, it's just a single linter.

Reply:
Fixed